.obs/chartfile/operator/values.yaml

```diff
@@ -10,9 +10,9 @@ seedImage:
   imagePullPolicy: IfNotPresent
 
 channel:
-  name: "sle-micro-channel"
-  image: "registry.suse.com/rancher/elemental-channel"
-  tag: "%VERSION%"
+  name: "sle-micro-5.5"
+  image: "%%IMG_REPO%%/rancher/elemental-channel/sle-micro"
+  tag: "5.5"
```

Comment:
This should be set to Micro 6.0 as soon as we get images available in registry. Meanwhile we should use the images we already have available.